internal/fileutils/fileutils.go

```diff
@@ -373,8 +381,7 @@ func PrependToFile(filepath string, data []byte) error {
 // AmendFile amends data to a file, supports append, or prepend
 func AmendFile(filePath string, data []byte, flag AmendOptions) error {
 	switch flag {
-	case
-		AmendByAppend, AmendByPrepend:
+	case AmendByAppend, AmendByPrepend:
```

Comment:
This and the change at line 52 were also formatting changes that I stopped trying to "unfix".